src/main/java/svc/data/citations/datasources/tyler/TylerViolationStatus.java

```diff
@@ -0,0 +1,37 @@
+package svc.data.citations.datasources.tyler;
+
+import svc.models.VIOLATION_STATUS;
+
+public enum TylerViolationStatus {
```

Comment:
It's converting between the internal api enum and the external "tyler" string. There should be a lot more cases in here, for each status tyler can return that we want to map to our API enums.